app/controllers/ActivationController.scala

```diff
@@ -22,15 +22,15 @@ import connectors.ApplicationClient
 import connectors.UserManagementClient.{ TokenEmailPairInvalidException, TokenExpiredException }
 import helpers.NotificationType._
 import security.Roles._
-import security.SignInUtils
+import security.SignInService
 
 import scala.concurrent.Future
 
-object ActivationController extends ActivationController {
+object ActivationController extends ActivationController(ApplicationClient) {
   val http = CSRHttp
```

Comment:
Is there a reason you elected to use constructor injection over the cake-y type pattern i'd say we "normally" use?